pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -241,6 +241,29 @@ func (r *ReconcileOperConfig) Reconcile(request reconcile.Request) (reconcile.Re
 		Name:     names.APPLIED_NAMESPACE,
 	})
 
+	// Add operator.openshift.io/v1/network to relatedObjects for must-gather
+	relatedObjects = append(relatedObjects, configv1.ObjectReference{
+		Group:    "operator.openshift.io",
+		Resource: "networks",
+		Name:     "cluster",
+	})
+
+	// Add NetworkPolicy, EgressFirewall, EgressIP, for must-gather
+	relatedObjects = append(relatedObjects, configv1.ObjectReference{
+		Group:    "networking.k8s.io",
+		Resource: "NetworkPolicy",
+	})
```

Comment:
This is not actually needed to prevent objects from being deleted; we only delete objects when the `ObjectReference` points to a _specific_ object, not when it points to an entire class of objects.

Reply:
You're right, I completely overlooked that. Thanks for spotting it.